README.md

```diff
@@ -48,7 +48,7 @@ No modules.
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| target\_arn | The ARN of a target (usually an SNS topic) that should be notified when an IAM or SSO user is created (e.g. arn:aws:sns:us-east-1:012345678901:my-sns-topic). | `string` | n/a | yes |
+| target\_arn | The ARN of a target (usually an SNS topic) that should be notified when certain IAM/SSO user or group events occur (e.g. arn:aws:sns:us-east-1:012345678901:my-sns-topic). | `string` | n/a | yes |
```

Comment:
I don't like the non-specific "certain IAM/SSO user or group events occur".  Can we update this using the language from the `aws_cloudwatch_event_rule.this` resource's `description` field?

Reply:
No problem- how does https://github.com/cisagov/new-user-alert-tf-module/pull/25/commits/4e33b3bf169c68663dbf235870e7ae42eb32d51c look to you?

Reply:
Looks groovy, man!